testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc="nodejs bash"
+makedepends="linux-headers"
```

Comment:
add as much libs as possible, we prefer to link dynamically. ie lz4-dev and http-parser-dev...
Dont forget to add nodejs to makedepends (if its needed).